controllers/applicationvolumereplication_controller_test.go

```diff
@@ -269,3 +269,166 @@ var _ = Describe("ApplicationVolumeReplication controller", func() {
 		})
 	})
 })
+
+var _ = Describe("isManifestInAppliedState", func() {
```

Comment:
Remove this.  It is already set above

Reply:
Not sure about the intent of this comment. Is the desire to have a single `Describe` block for all the AVR controller tests, rather than having a second `Describe` block for `IsManifestInAppliedState`? Or should the second `Describe` block for `IsManifestInAppliedState` be nested within the overall `Describe` block for AVR controller tests?

Reply:
Yeah, that's what I meant.  Just use one `var _ = Describe...`, which is already above.

Reply:
This wasn't quite as straightforward as I expected. Rolling that `Describe` into the general AVR tests created some BeforeEach issues. I left this as a TODO with my name on it on `d41ab23fee18f7e5e248be22830fc12385e9075e` so I can fix it when I'm back. We can just make an issue for this if nobody objects.